.github/ISSUE_TEMPLATE/add-orbit-chain-request.yml

```diff
@@ -59,6 +59,7 @@ body:
       label: Explorer URL
     validations:
       required: true
+      
   - type: input
```

Comment:
nit, one more suggestion for parent chain ID. Maybe to make it more user friendly we could do something like this:
```
- type: dropdown
    id: parent-chain-id
    attributes:
      label: Parent Chain ID
      options:
        - 1
        - 42161
        - 11155111
        etc
```
Or even show network names and map them to chain IDs on our end? Unfortunately I don't think GH can map them for us like this